src/scss/components/optional/_page-banner.scss

```diff
@@ -1,69 +1,140 @@
+*
+* Documentation:
+* https://fozzie.just-eat.com/styleguide/ui-components/page-banner
+*/
+
+//Vars to use for IE homepage banner rays positioning fix
+$spaceForColouredRays: 20px;
+$spaceForColouredRays-tiny: 60px;
+$spaceForColouredRays-mid: 164px;
+$spaceForColouredRays-wide: 148px;
+
 
 @mixin pageBanner() {
 
-    .c-pageBanner {
-        background-color: $white;
-        margin-bottom: -#{spacing(x2)};
-        max-height: 300px;
-        min-height: 100px;
-        overflow: hidden;
-        position: relative;
+   .c-pageBanner {
+       background-color: $white;
+       max-height: 300px;
+       min-height: 100px;
+       overflow: hidden;
+       position: relative;
 
-        @include media('>=wide') {
-            max-height: 400px;
-        }
+       @include media('>=wide') {
+           max-height: 400px;
+       }
+   }
 
-        // Prevent IE from applying these styles as it can't correctly position the rays
-        @supports (position: absolute) {
-            &:after {
-                background-image: inline('#{$img-path}/decoration/rays--white--small.svg');
-                background-position: bottom left;
-                background-repeat: no-repeat;
-                bottom: -1px;
-                content: '';
-                height: 100%;
-                left: 0;
-                position: absolute;
-                width: 100%;
-                z-index: zIndex(low);
-
-                @include media('>=mid') {
-                    background-image: inline('#{$img-path}/decoration/rays--white.svg');
-                }
-            }
-        }
-    }
+   .c-pageBanner--wide {
+       max-height: 600px;
 
-        // any elements below the banner will be moved up, over the bottom part of the banner
-        .c-pageBanner--negativeBottom--aboveMid {
-            @include media('>=mid') {
-                margin-bottom: -115px;
-            }
+       &.c-pageBanner--withRays--coloured {
+           .c-pageBanner-img {
+               margin-bottom: $spaceForColouredRays;
 
-            @include media('>=wide') {
-                margin-bottom: -150px;
-            }
-        }
+               @include media('>=tiny') {
+                   margin-bottom: $spaceForColouredRays-tiny;
+               }
+
+               @include media('>=mid') {
+                   margin-bottom: $spaceForColouredRays-mid;
+               }
+
+               @include media('>=wide') {
+                   margin-bottom: $spaceForColouredRays-wide;
+               }
+           }
+       }
+   }
+
+   .c-pageBanner--narrow {
+       max-height: 300px;
+
+       @include media('>=huge') {
+           max-height: 370px;
+       }
+       .c-pageBanner-img {
+           @include media('>=mid') {
+               margin-top: -8%;
+           }
+       }
+   }
+
+   .c-pageBanner--withRays--white {
+       // Prevent IE from applying these styles as it can't correctly position the rays
+       @supports (position: absolute) {
```

Comment:
Yes, this avoids IE  ¯\_(ツ)_/¯

Reply:
I don't understand this either. Do these styles get ignored in IE11?

Reply:
Ah I see, your using the `@supports` - is this the best way tho, what happens when IE11 starts supporting that?

Reply:
would it be better to use `@media all and (-ms-high-contrast: none), (-ms-high-contrast: active)` to target IE10/11 and hide the pseudo elm - just incase IE11 does start supporting supports - or they fix the bug(? seems like a bug as it does support absolute 😄).

Reply:
Sounds good to me, @ashleynolan @kevinrodrigues any objections?

Reply:
Actually - I get what's happening now.  Because IE11 doesn't support `@supports` (meta 💩) so then this won't get run.  It's incidental that it targets `position: absolute` – it could be anything and it would never run right?

IE11 will never support this feature so this should be safe to leave as is?

Reply:
Never say never :D I think if you update the comment so it explains that you're using `@supports` to bypass IE then that would be ok with me.

Reply:
If Microsoft go back and start adding new features to IE11 I'll go back and fix this myself 😉